code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -59,7 +59,7 @@
 	if(!proximity)
 		return
 
-	if(target.is_refillable()) //Something like a glass. Player probably wants to transfer TO it.
+	if(target.is_refillable() && is_drainable()) //Something like a glass. Player probably wants to transfer TO it.
```

Comment:
i'd say make this a new statement underneath, similar to l81. also shouldn't l81 be using is_drainable too?